apicommunication/transport.go

```diff
@@ -382,3 +383,57 @@ func ValidateRequest(r *http.Request, st storage.Store) (*storage.JiraInstallInf
 	}
 	return jii, nil
 }
+
+const kidValidationURL = "https://connect-install-keys.atlassian.com/"
+
+// ValidateInstallRequest attempts to validate new install method for jira
+func ValidateInstallRequest(r *http.Request, st storage.Store) error {
+	q := r.URL.Query()
+	queryJWT := q.Get("jwt")
```

Comment:
Minor comment, but per [Atlassian's doc on decoding JWT](https://developer.atlassian.com/cloud/jira/platform/understanding-jwt-for-connect-apps/) I think this can go:

> 1. Extract the JWT token from the request's authorization header (using a jwt query parameter is deprecated)

The same comment holds for `ValidateRequest` which is outside the scope of this patch.

Reply:
Yes, notice we fallback to header, turns out Atlassian is not as solid in their practice as they are in their docs (guess why this double check exists :p)

Reply:
Got it :-) Thanks